lab-khalid/lib/greet.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+module.exports = exports = {};
```

Comment:
You're only putting a single function on module.exports, so you don't need to make it an empty object to attach to.